CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.32.0]
+
+- Update compatibility table.
```

Comment:
IMO this is implicit so you don't need it, it's fine to leave the documentation changelog empty since it itself didn't change, isn't it?